test/tests/translateTest.js

```diff
@@ -571,6 +572,49 @@ describe("Zotero.Translate", function() {
 			assert.equal(pdf.attachmentLinkMode, Zotero.Attachments.LINK_MODE_IMPORTED_URL);
 			checkTestTags(pdf, true);
 		});
+		
+		it('child translators should not convert tags to canonical form', function* () {
```

Comment:
I know it's not done everywhere above, but let's try to keep these as sentences like they're intended:

`it('should not convert tags to canonical form in child translators')`